src/xdocs/writingjavadocchecks.xml.vm

```diff
@@ -558,11 +558,124 @@ JAVADOC -> JAVADOC [0:0]
         </td>
       </tr>
     </table>
-    </section>
 
-    <section name="Boolean flag indicating the presence of unclosed HTML tags">
-      Not implemented yet, will be named "hasUnclosedTag". See <a href="https://github.com/checkstyle/checkstyle/issues/3311">Github Issue #3311</a>.
-      For expected behavior please read <a href="#Javadoc_parser_behavior_for_current_HTML_version_and_new_HTML_version">Javadoc parser behavior</a> section.
+    <p>
+        Checks can also be configured to log violation on encoutering non-tight HTML tags.
+        <tt>violateExecutionOnNonTightHtml</tt> property can be used for this purpose in the checks
+        that support it. A custom check needs to extend <tt>AbstractJavadocCheck</tt> for having
+        this functionality readily available. Do note that a check which has this property set to
+        true will immediately log a violation upon finding a non-tight HTML tag and proceed to the
```

Comment:
`log a violation upon finding a non-tight HTML tag` => `log violations upon finding any non-tight HTML tags`
It will log all, not just 1.

Reply:
Sorry my bad. Corrected. Thank you.